run.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 cd /usr/src/app
 
-bundle exec rake db:create db:migrate
```

Comment:
@starswan if this is removed at what step will migrations be done? What is the reason for _not_ doing it in `run.sh`